torch/_inductor/codegen/common.py

```diff
@@ -489,20 +489,26 @@ def generate(
         buffer: IndentedBuffer,
         expr: typing.Union[str, CSEVariable],
         write=True,
+        expression=True,  # If False, we assume it's a statement and returns nothing
```

Comment:
bike-shedding - should this be `assignment`?

Reply:
I went with the PL name expression/statement, but I do prefer your name, yep.